### ISSUE TITLE
[Issue] Adding export to wrong card

### ISSUE THREAD

**Issue by msw93** [animebook/animebook.github.io#67]
This is a user based error but sometimes I click the "add context" button by accident or click the add to anki and then "add context "button in the wrong order and I replace the fields in the incorrect card, my version on anki does not allow me undo these changes. In this case I have to find the previous sentence (usually from a previous episode) and remake the card.

Not sure if anyone else has had these issues? Also, I am not sure if there is clean solution this problem either...

**Comment by soamsy** [comment]
Yeah, there's not a clean solution to this. I currently have it set up so that if the card you're trying to update was created more than 2 days ago, it will fail, claiming you need to create a card first. So at the very least, there's a safe guard if you're coming back to the video player after a couple days.

The problem comes from the fact I want you to be able to "retry" if you grab the wrong context, or click the wrong + button. Since you're able to retry, you're able to overwrite by accident later.

**Comment by msw93** [comment]
Sorry I know this was closed. But what do you think about changing the time out to 8 hours? 

In this case it would almost always be a different session and would prevent next day overwrites.

**Comment by soamsy** [comment]
I've been using `added:2` in the anki search query. It looks like I could knock it down to `added:1` for 1 day, but I don't know if that means "added in the last 24 hours" or "added after 4:00 am today (or whenever your Anki starts the next day)".
https://docs.ankiweb.net/searching.html#added
Documentation doesn't make it clear, but if it's the latter, does that mean a card added at 3:59 am can't be used a minute later? I'd need to test to figure that out.

**Comment by soamsy** [comment]
There's also the is:new option, which would prevent you from updating cards you've already reviewed. I'm ok with adding that, if that would help.

**Comment by msw93** [comment]
Yes the is:new would remove the most annoying case (when I do this the next day by accident)

thank you!
